app/screens/wallet/Backup.js

```diff
@@ -0,0 +1,187 @@
+
+const Actionable = css`
+  &:hover {
+    opacity: 0.8;
+  }
+  &:active {
+    opacity: 0.6;
+  }
+`;
+
+const Wrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 50px;
+`;
+
+const Header = styled.span`
+  font-size: 31px;
+  font-weight: bold;
+  line-height: 42px;
+  color: ${smColors.lighterBlack};
+  margin-bottom: 20px;
+`;
+
+const BaseText = styled.span`
+  font-size: 16px;
+  font-weight: normal;
+  line-height: 22px;
+  color: ${smColors.lighterBlack};
+`;
+
+const BoldText = styled(BaseText)`
+  font-weight: bold;
+`;
+
+const GrayText = styled(BaseText)`
+  color: ${smColors.darkGray};
+`;
+
+const Separator = styled.div`
+  border-bottom: 1px solid ${smColors.borderGray};
+  margin: 12px 0;
+`;
+
+const BackupBoxesWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 12px 0;
+`;
+
+// $FlowStyledIssue
+const ActionLink = styled(BaseText)`
+  user-select: none;
+  color: ${({ theme }) => (theme === 'orange' ? smColors.orange : smColors.green)};
+  cursor: pointer;
+  ${Actionable}
+`;
+
+// $FlowStyledIssue
+const BackupBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  ${({ borderColor }) =>
+    borderColor &&
+    `
+    border: 1px solid ${borderColor};
+    
+    `}
+  /* max-width: 468px; */
+  height: 384px;
+  flex: 1;
+  &:first-child {
+    margin-right: 16px;
+  }
+  border-radius: 2px;
+  padding: 24px 18px;
+`;
+
+const BackupTopWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+`;
+
+const SecurityLogo = styled.img`
+  height: 66px;
+  width: 54px;
+  margin-right: 30px;
+`;
+
+const BackupHeader = styled.span`
+  font-size: 24px;
+  font-weight: bold;
+  line-height: 33px;
+  color: ${smColors.darkGray};
+`;
+
+const BackupTitleWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`;
+
+const Recommended = styled.div`
+  align-self: flex-end;
+  padding-left: 24px;
+`;
+
+type Props = {};
+
+class Backup extends Component<Props> {
+  render() {
+    return (
+      <Wrapper>
+        <Header>Wallet Backup</Header>
+        <BaseText>Your wallet file is encrypted with your passphrase on your computer, but We recommend that you’ll backup your wallet for additional security.</BaseText>
+        <Separator />
+        <BoldText>How would you like to backup your wallet?</BoldText>
+        <BackupBoxesWrapper>
+          {this.renderBackupBox('file')}
+          {this.renderBackupBox('mnemonic')}
+        </BackupBoxesWrapper>
+        <ActionLink onClick={this.learnMoreAboutSecurity}>Learn more about wallet security and backup</ActionLink>
+      </Wrapper>
+    );
+  }
+
+  renderBackupBox = (mode: 'file' | 'mnemonic') => {
+    const shieldLogo = mode === 'file' ? shieldIconGreenOne : shieldIconOrangeTwo;
```

Comment:
same evaluation is repeated multiple time

Reply:
assigned to const